sal_interfaces/Watcher/Watcher_Events.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
+<SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
+    <Enumeration>AlarmSeverity_None,AlarmSeverity_Warning,AlarmSeverity_Serious,AlarmSeverity_Critical</Enumeration>
+
+<SALEvent>
+    <Subsystem>Watcher</Subsystem>
+    <Version>1.0.0</Version>
+    <Author></Author>
+    <EFDB_Topic>Watcher_logevent_heartbeat</EFDB_Topic>
+    <Alias>heartbeat</Alias>
+    <Explanation>http://sal.lsst.org</Explanation>

```

Comment:
probably should remove the URL since it goes nowhere.

Reply:
Thank you! I thought it was required.